holidays/countries/india.py

```
@@ -501,8 +540,7 @@ def _populate_subdiv_cg_public_holidays(self):
 
     # Delhi.
     def _populate_subdiv_dl_public_holidays(self):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_populate_subdiv_dl_public_holidays`

Add return type annotation: `None`

(ANN202)